response_operations_ui/forms.py

```diff
@@ -495,3 +512,15 @@ class BannerEditForm(FlaskForm):
     )
     banner_id = StringField("id")
     delete = BooleanField("Delete banner", default=False)
+
+
+class Form(FlaskForm):
+    def validate(self):
+        return "option" in self.data and self.data["option"]
+
+
+class OptionsForm(Form):
```

Comment:
This has a generic name, but is really a form for a specific page.  Something like 'MyAccountForm' or something like that would be better I think.